### DISCUSSION TITLE
Auto generated contract by garaga gen command does not recognises as contract

### DISCUSSION THREAD

**Discussion by erdoganishe** [keep-starknet-strange/garaga#217] [Q&A] [answered]
I have a snarkjs generated artifacts (verification_key.json, public_inputs.json, proof.json, etc).

I generate cairo contract using `garaga gen --system groth16 --vk verification_key.json`, which created 3 files, as expected, but when I trying to declare project (garaga declare-project --project-path my/project/path --env-file .secrets --network sepolia --fee strk), i get an error:
class hash not found - deploying
Error during declaration: Client failed with code 20. Message: Contract not found.

I`m sure that path to project is correct.
I didn`t changed autogenerated cairo files too. 

Thank you in advance.

**Comment by feltroidprime** [comment, marked as answer]
Hi!! I've ran into a similar problem. 
The temp fix is to simply re-run the command and you will get the class hash. 
 The flow of this command needs to be fixed, it is working but it is a false positive error.

**Reply by feltroidprime** [reply]
Alternatively you can use [starkli](https://github.com/xJonathanLEI/starkli) which is now compatible with Cairo 2.8.2 and therefore Garaga.
